.github/workflows/workflow.yml

```
@@ -11,37 +11,44 @@ jobs:
-        if: ${{ !steps.release-check.outputs.IS_RELEASE }}
+        if: ${{ !steps.fork-check.outputs.IS_FORK && !steps.release-check.outputs.IS_RELEASE }}
       - uses: actions/setup-node@v6
         with:
           node-version: 22
-        if: ${{ !steps.release-check.outputs.IS_RELEASE }}
+        if: ${{ !steps.fork-check.outputs.IS_FORK && !steps.release-check.outputs.IS_RELEASE }}
       # Use npm@10 on Node 22+ due to https://github.com/npm/cli/issues/8489
       - name: Setup npm version
         run: npm install -g npm@10
-        if: ${{ !steps.release-check.outputs.IS_RELEASE && matrix.node-version == '22' }}
+        if: ${{ !steps.fork-check.outputs.IS_FORK && !steps.release-check.outputs.IS_RELEASE && matrix.node-version == '22' }}
```

actionlint (1.7.12)

[error] property "node-version" is not defined in object type {}

(expression)